notebooks/ivf_flat_example.ipynb

```diff
@@ -21,7 +21,7 @@
   },
```

Comment:
<p>This error here seems alarming. We should have <code>extend</code> available? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuvs/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
Ah, actually I need to run the notebooks again. That was fixed awhile ago